no-computer2.md

```diff
@@ -0,0 +1,81 @@
+# No computer project 2
+Legislation
+
+## The user and a language
+This section describes who the project would serve and why a language might be a
+good way to meet their needs.
+
+
+
+### What's the need?
+_What need is met by your idea? Who are you helping? What is that person's
+experience like now? What would their experience be like if you could help 
+them?_
+The need is to have a uniform way of writing laws. Without a system in place a persons experience would be scattered and difficult to understand and complex. The experience with a regulated system is much better.
+
+
+### Why a language?
+_Why is a DSL appropriate for your user(s)? How does it address the need?_
+A DSL is appropriate because we need to write laws in language but without guidelines the above problems would run rampant, with regulation these problems are mitigated.
+
+
+### Why you?
+_What excites you about this idea? How did you come up with it?_
+This idea is cool because its necessary, but I actually think that this system is pretty archiac and difficult for the layman to understand. I came up with this idea when I was reading some legislation for my politics class.
+
+
+### Domain
+_Describe the project's domain in five words._
+Old white dudes writing laws.
```

Comment:
I don't really see why this is necessary. Plus you didn't specify American legislature. An african countries legislature would most definitely not be white most likely. I do like that you involved the action this time though.